### DISCUSSION TITLE
Support for fieldset and legend for radio input with @tailwindcss/forms

### DISCUSSION THREAD

**Discussion by apoorvmote** [tailwindlabs/tailwindcss#3829] [Ideas]
I don't use select option anymore because it requires two clicks. I have converted all of my selects into radio.

Best use case for fieldset is to show that all radio buttons belong to single group field. And have legend to show the field name.

I am using @tailwindcss/forms plugin and currently I am using variable paddings to show that these radio buttons belong to one group. I would like to have support for legend and fieldset to group related radios together.

**Comment by Badlapje** [comment]
I would also like to request this.  From an a11y-perspective fieldsets are a must when dealing with multiple options.  See: https://a11y-style-guide.com/style-guide/section-forms.html#kssref-forms-radio-buttons
